src/components/ConnectExchangeView/ConnectExchangeView.js

```diff
@@ -11,6 +12,7 @@ import FAQ from "../FAQ";
 /**
  * @typedef {Object} DefaultProps
  * @property {Function} onClose
+ * @property {ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>} onSearch Search callback
```

Comment:
Maybe you can use React.ChangeEventHandler to avoid // eslint-disable-next-line no-unused-vars

Reply:
sure!